docs/source/author/topics.rst

```diff
@@ -186,9 +185,9 @@ The Markdown file containing the content for the lesson:
 
 .. note::
 
-  If a lesson includes programming challenges, and/or learning outcomes, then
-  the corresponding configuration and content files may also need to be added
-  or updated.
+  If a lesson includes programming challenges, Computational Thinking links,
+  and/or learning outcomes, then the corresponding configuration and content
+  files may also need to be added or updated.
```

Comment:
I think this note should stay as it was an instead the CT links should be added to the flowchart, since CT links have an actual content file, not just extra lines in a config file (like LOs), and are not referenced anywhere else (where as programming challenges have their own section for how to add them)